test/models/test_dynamic_port_binding.rb

```diff
@@ -455,6 +455,42 @@ module Models
                 end
             end
 
+            describe "injecting another instantiation class" do
+                before do
+                    @port = DynamicPortBinding.new(flexmock, flexmock,
+                                                   output: true, port_resolver: nil)
+                end
+                it "raises if klass is not a subclass of BoundOutputReader" do
+                    foo = Class.new
+                    assert_raises ArgumentError do
+                        DynamicPortBinding::BoundOutputReader.new(
+                            "bla", flexmock, @port, klass: foo
+                        )
+                    end
+                end
+
+                it "instantiates using BoundOutputReader when klass is not passed" do
+                    b = DynamicPortBinding::BoundOutputReader.new("bla", flexmock, @port)
+                    result = b.instanciate(flexmock)
+                    assert(
+                        result.instance_of?(Syskit::DynamicPortBinding::BoundOutputReader)
```

Comment:
I think rubocop wants `kind_of?` (generally speaking, I use kind_of? so please stick with that)

Reply:
Plus, I believe minitest has `assert_kind_of`